extensions/SampleExtension/Extension/DeveloperIdProvider.cs

```diff
@@ -13,7 +12,7 @@ internal sealed class DeveloperIdProvider : IDeveloperIdProvider
     public string DisplayName => $"Sample {nameof(DeveloperIdProvider)}";
 
 #pragma warning disable CS0067 // The event 'DeveloperIdProvider.Changed' is never used
-    public event TypedEventHandler<IDeveloperIdProvider, IDeveloperId> Changed;
+    public event TypedEventHandler<IDeveloperIdProvider, IDeveloperId>? Changed;
```

Comment:
NIT: Add a comment explaining what the developer gets if they use Changed.

Reply:
The entire provider is stubbed out, I just fixed the signature, so I'm not going to mess with it. If @vineeththomasalex or @ssparach wants to turn this into a better sample, they are welcome to do so in a subsequent PR. I would also strongly encourage that, so we have a good DeveloperId provider sample, but I don't want to go adding documentation for one part when the entire class is a stub.